docs/use-cases/organizational-access-control.mdx

```diff
@@ -0,0 +1,36 @@
+---
+
+<!--todo: description --->
+
+---
+
+# Organizational Access Control
+
+## Abstract
+
+Typically, organizations employ role-based access control (RBAC) to manage access to restricted resources.
+DAOs often do not have such a mechanism, but both types of organizations suffer from similar problems.
+Some of them are:
+- Members control some entity with the access (e.g their account) and not the access directly;
```

Comment:
I wanted to say that in traditional organisations when you lose your account – you lose the access. And with Rarimo you don't have an account as it is, because you can login with different identities (profiles) using the access card (credentials) + you store this credentials locally in your snap

Reply:
Doesn't seem like something important. Let's emphasize the anonymous proofs of membership